README.md

```diff
@@ -38,26 +49,27 @@ Python 3.8+
 poetry add noteable-origami
 ```
 
-
 ### Pip
+
 ```shell
 pip install noteable-origami
 ```
+
 <!-- --8<-- [end:install] -->
 
 <!-- --8<-- [start:start] -->
-## Getting Started
 
-Get your API token from Noteable Within user settings.
-Within user settings, go to the API Token page, and generate a new token. 
+## Getting Started
 
-Copy the value
+Get your API token from Noteable within user settings.
+Within user settings, go to the API Token page, and generate a new token.
 
 ```python
 from origami.client import NoteableClient
 
-token = 'ey...' # Your user API token
+token = ''  # Your API token from Noteable
 async with NoteableClient(api_token=token) as client:
+    # Establish a connection to the realtime API
```

Comment:
The connection establishment happens as part of the async context manager entry method and not with the `ping_rtu` which is what the comment suggests to me, at least. Perhaps we should move the comment above the `async with NoteableClient`?

Reply:
Moved it up!